src/mod_muc_room.erl

```diff
@@ -3295,13 +3295,12 @@ change_config(Config, StateData) ->
     case {(StateData#state.config)#config.persistent,
 	  Config#config.persistent}
 	of
-      {_, true} ->
-	  mod_muc:store_room(NSD#state.server_host,
-			     NSD#state.host, NSD#state.room, make_opts(NSD));
-      {true, false} ->
-	  mod_muc:forget_room(NSD#state.server_host,
-			      NSD#state.host, NSD#state.room);
-      {false, false} -> ok
+      {_, _} ->
+        mod_muc:forget_room(NSD#state.server_host,
+          NSD#state.host, NSD#state.room),
+	      mod_muc:store_room(NSD#state.server_host,
+          NSD#state.host, NSD#state.room, make_opts(NSD));
+        {false, false} -> ok
```

Comment:
wouldn't `{_, _}` catch everything before you're able to reach `{false, false}`? Or am I reading this wrong

Reply:
Whoops, wrong order, you're right